src/table/hooks/__tests__/use-column-widths.spec.ts

```diff
@@ -13,7 +13,9 @@ import useColumnWidths from "../use-column-widths";
 jest.mock("@qlik/nebula-table-utils/lib/hooks");
 
 describe("use-column-widths", () => {
-  let mockedUseMeasureText: jest.MockedFunction<(props: UseMeasureTextProps) => MeasureTextHook>;
+  let mockedUseMeasureText: jest.MockedFunction<
+    (props: UseMeasureTextProps, options?: { maxNbrLinesOfText?: number }) => MeasureTextHook
+  >;
```

Comment:
@cbt1 I guess we could export smarter types here, I'd say `UseMeasureTextProps` isn't really correct either

Reply:
Hmm not sure, what is wrong with `UseMeasureTextProps`?

Reply:
Ah yes the naming could probably be improved